apps/finance/app/src/components/Transfers.js

```diff
@@ -181,37 +218,74 @@ class Transfers extends React.Component {
 }
 
 const Header = styled.div`
-  display: flex;
-  justify-content: space-between;
-  flex-wrap: nowrap;
   margin-bottom: 10px;
+
+  ${breakpoint(
+    'medium',
+    `
+      display: flex;
+      justify-content: space-between;
+      flex-wrap: nowrap;
+  `
+  )};
 `
 
-const Filters = styled.div`
-  display: flex;
-  flex-wrap: nowrap;
+const Filters = styled(animated.div)`
+  display: grid;
+  grid-template-columns: 50% 50%;
+  margin: 0 20px 10px 20px;
+
+  ${breakpoint(
+    'medium',
+    `
+      display: flex;
+      flex-wrap: nowrap;
+      margin: 0;
+    `
+  )};
 `
 
 const FilterLabel = styled.label`
-  display: flex;
-  flex-wrap: nowrap;
-  align-items: center;
-  white-space: nowrap;
+  ${breakpoint(
+    'medium',
+    `
+      display: flex;
+      flex-wrap: nowrap;
+      align-items: center;
+      white-space: nowrap;
+    `
+  )};
 `
 
 const Title = styled.h1`
-  margin-top: 10px;
-  margin-bottom: 20px;
+  margin: 20px 20px 10px 20px;
   font-weight: 600;
+  display: flex;
+  justify-content: space-between;
+
+  ${breakpoint(
+    'medium',
+    `
+      margin: 10px 30px 20px 0;
```

Comment:
Hmm, is the right margin here necessary?

Actually, I was thinking of increasing the top margin to align it with the filters on larger screens. Thoughts @bpierre @AquiGorka @jounih?

Currently:

![image](https://user-images.githubusercontent.com/4166642/51853000-91e65980-2327-11e9-96f6-0ee53a0f39ff.png)

With more top margin (`30px`):

![image](https://user-images.githubusercontent.com/4166642/51853006-96ab0d80-2327-11e9-97d3-d8f81f5a4614.png)

Reply:
Can we push this for the backlog?

Reply:
Proposed fix: https://github.com/aragon/aragon-apps/pull/767